### DISCUSSION TITLE
Server resources

### DISCUSSION THREAD

**Discussion by sbwalker** [oqtane/oqtane.translations#24] [General]
@hishamco in the English folder in this repo there are subfolders for Oqtane.Client and Oqtane.Server - are these supposed to be copied to their respective locations in the Oqtane framework project ( ie. Oqtane.Client -> Oqtane.Client  and Oqtane.Server -> Oqtane.Server ) ? Or should the entire English folder and its contents be copied to the Oqtane.Client\Resources folder?

**Comment by hishamco** [comment]
All the folders should go inside `Oqtane.Client` that's why we added the path here. FYI the English resources are just there to start create a new translation without using `Oqtane.ResxExtractor` tool
